<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chrono</class>
 <widget class="QWidget" name="Chrono">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:transparent;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="m_pMsDelta">
     <property name="text">
      <string>Millisecond delta</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_pEpoch">
     <property name="text">
      <string>Epoch (ms)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_pTime">
     <property name="text">
      <string>Timestamp (s)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_pDateTime">
     <property name="text">
      <string>Date and time</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>181</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
